bliss/models/binary.py

```diff
@@ -31,6 +31,7 @@ def __init__(
         spatial_dropout: float,
         dropout: float,
         optimizer_params: dict = None,
+        catalog_tag: str = "galaxy_bools",
```

Comment:
I gather this is to accommodate "galaxy_bools" and "lensed_galaxy_bools". This change makes it hard to understand the code since it requires knowing this context.
It would be better to have a separate Module for the lensed_galaxy_bools. It's better to violate the DRY principle a bit for clarity and to keep things disentangled.